Src/Generic/Dummy/ApplicationState.swift

```diff
@@ -9,9 +9,18 @@
 import Foundation
 
 struct ApplicationState {
```

Comment:
As this isn't super dummy anymore, let's move it :D

Reply:
true ✔️